docs/config.json

```
@@ -595,7 +595,25 @@
           "slug": "/application-access/getting-started/"
         },
         {
-          "title": "Access Guides",
+          "title": "Securing Cloud APIs",
```

```suggestion
          "title": "Securing Cloud Consoles",
```

What do you think about saying "consoles" instead? To me, it's more obvious that this is about access to the major hyperscalers and not any arbitrary cloud API.

---

@zmb3 Would "Securing Cloud Provider APIs" be clearer?

---

I personally think consoles is better. Azure calls it a Portal but it's clear that what a console is. API feels a little limiting.

---

The only console we can protect with the Application Service is the AWS console. For Azure and Google Cloud, the Application Service protects CLI tools by proxying requests to the cloud provider's API. I think "Securing Cloud Provider APIs" may be the most precise description.